src/components/CarouselSlide.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div v-show="visibleSlide === index" class="carousel-slide">
```

Comment:
no need to create an extra component for one line element

Reply:
okay done